src/main/resources/templates/fragments/chaptercard.html

```diff
@@ -6,7 +6,7 @@
 <body>
     <a th:fragment="chaptercard(index, text, url)" class="chaptercard"
        th:classappend="${url == null}? 'unavailable'"
-       th:href="${url == null}? '#' : ${'/download/' + url}"
+       th:href="${url == null}? '#' : ${'//' + baseUrl + '/download/' + url}"
```

Comment:
I don't think we need '/download/' here. We can omit it in the docker-compose fileserver service as well. If we decide to add it later it can be part of baseUrl, env vars, and nginx handling.

Reply:
Sure thing. I'll address the other comment related to this too.

Reply:
I suggest  we should provide a complete url from the backend like "http://downloadserver.fetcher/en/ulb/..../audio.mp3"
and not to append the baseUrl here